### PR TITLE
Update ScummVM to 2.7.0

### PR DESCRIPTION
**Pull request by AsciiWolf**
Do not merge before the official 2.7.0 release.

### CODE REVIEW

**Comment by AsciiWolf** [comment]
`• attribute-invalid     : <release> timestamp is in the future`

Ugh, I hate strict AppStream metadata checks. Anyway, it built fine and will most likely work without issues. I will test the new build after ScummVM 2.7.0 actually is released.

File: org.scummvm.ScummVM.appdata.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright 2020-2022 The ScummVM Team -->
+<!-- Copyright 2020-2023 The ScummVM Team -->
 <component type="desktop">
   <id>org.scummvm.ScummVM.desktop</id>
   <metadata_license>CC0-1.0</metadata_license>
```
Comment (lotharsm) [inline org.scummvm.ScummVM.appdata.xml:5]: Can you please bump the license to GPLv3+? We had a change in licensing a while ago :smiley:

Comment (AsciiWolf) [reply org.scummvm.ScummVM.appdata.xml:5]: https://github.com/flathub/org.scummvm.ScummVM/pull/29/files

@lotharsm Any chance you could also fix project_license in the [upstream AppData file](https://github.com/flathub/org.scummvm.ScummVM/blob/master/org.scummvm.ScummVM.appdata.xml#L6)? Thanks!

Comment (AsciiWolf) [reply org.scummvm.ScummVM.appdata.xml:5]: I have submitted the upstream fix myself: https://github.com/scummvm/scummvm/pull/4718

**Comment by lotharsm** [comment]
Can you flip the magic switch? ;-)

**Comment by AsciiWolf** [comment]
bot, build

**Comment by AsciiWolf** [comment]
I may have found a regression. :-(

It looks like that Broken Sword: The Shadow of the Templars is broken in 2.7.0 (works fine in 2.6.2):

screenshot

Sadly nothing useful in the game log (even with debuglevel set):

```
User picked target 'sword1' (engine ID 'sword1', game ID 'sword1')...
   Looking for a plugin supporting this target... Broken Sword: The Shadow of the Templars
MetaEngine: Broken Sword: The Shadow of the Templars 	 matched to 	 Engine: (null)
Engine: (null) matched to MetaEngine: Broken Sword: The Shadow of the Templars
WARNING: Broken Sword: The Shadow of the Templars failed to instantiate engine: Game data not found (target 'sword1', path '/home/asciiwolf/Dokumenty/scummvm/Broken Sword')!
```

All the other games work fine.

**Comment by AsciiWolf** [comment]
Anyway, I am going to merge this PR and report this regression to upstream. Hopefully there are no other regressions. :-)

**Comment by lotharsm** [comment]
Hi,

> It looks like that Broken Sword: The Shadow of the Templars is broken in 
> 2.7.0 (works fine in 2.6.2):
> 

indeed we switched over to our more sophisticated checksum-based 
detection for Broken Sword for 2.7.0.

Which version of the game are you running exactly? It is possible that 
your version is missing from the detection tables despite our efforts to 
cover them all :/

**Comment by lotharsm** [comment]
> It looks like that Broken Sword: The Shadow of the Templars is broken in 
> 2.7.0 (works fine in 2.6.2):
> 

Ah, could you please try to remove the game from the launcher and re-add 
it again?

**Comment by AsciiWolf** [comment]
Ah! My version is an old Czech version of the game from one of our game magazines from 2004 that I modified to work with ScummVM years ago. :-) My Broken Sword II: The Smoking Mirror copy is from the same magazine and it works fine in ScummVM 2.7.0.

**Comment by AsciiWolf** [comment]
This is how the files structure looks like:

```
Broken Sword
├── Clusters
│   ├── compacts.clu
│   ├── general.clu
│   ├── ireland.clu
│   ├── maps.clu
│   ├── paris1.clu
│   ├── paris2.clu
│   ├── paris3.clu
│   ├── paris4.clu
│   ├── scotland.clu
│   ├── scripts.clu
│   ├── spain.clu
│   ├── swordres.rif
│   ├── syria.clu
│   ├── text.clu
│   └── train.clu
├── Music
│   ├── MARQUET.wav
│   ├── RM3A.wav
│   ├── RM3B.wav
│   ├── RM3C.wav
│   ├── RM3D.wav
│   ├── RM4.wav
│   ├── RM5.wav
│   ├── RM6.wav
│   ├── RM7.wav
│   ├── RM8.wav
│   ├── SCM1B.wav
│   ├── SCM1.wav
│   ├── SCM11.wav
│   ├── SCM16.wav
│   ├── SCM2.wav
│   ├── SCM3.wav
│   ├── SCM4.wav
│   ├── SCM5.wav
│   ├── SCM6.wav
│   ├── SCM7.wav
│   ├── SCM8.wav
│   ├── SPM2.wav
│   ├── SPM3.wav
│   ├── SPM4.wav
│   ├── SPM5.wav
│   ├── SPM6B.wav
│   ├── SPM6.wav
│   ├── 1M10.wav
│   ├── 1M11.wav
│   ├── 1M12.wav
│   ├── 1M13.wav
│   ├── 1M14.wav
│   ├── 1M15.wav
│   ├── 1M16.wav
│   ├── 1M17.wav
│   ├── 1M18.wav
│   ├── 1M19.wav
│   ├── 1M2.wav
│   ├── 1M20.wav
│   ├── 1M21.wav
│   ├── 1M22.wav
│   ├── 1M23.wav
│   ├── 1M24.wav
│   ├── 1M25.wav
│   ├── 1M26.wav
│   ├── 1M27.wav
│   ├── 1M28A.wav
│   ├── 1M28.wav
│   ├── 1M29.wav
│   ├── 1M3.wav
│   ├── 1M30.wav
│   ├── 1M31.wav
│   ├── 1M32.wav
│   ├── 1M34.wav
│   ├── 1M35.wav
│   ├── 1M4.wav
│   ├── 1M6.wav
│   ├── 1M7.wav
│   ├── 1M8.wav
│   ├── 1M9.wav
│   ├── 10M1.wav
│   ├── 10M2.wav
│   ├── 10M3.wav
│   ├── 10M4.wav
│   ├── 11M1.wav
│   ├── 11M2.wav
│   ├── 11M3.wav
│   ├── 11M4.wav
│   ├── 11M7.wav
│   ├── 11M8.wav
│   ├── 12M1.wav
│   ├── 2M1.wav
│   ├── 2M10.wav
│   ├── 2M11.wav
│   ├── 2M12.wav
│   ├── 2M13.wav
│   ├── 2M14.wav
│   ├── 2M15.wav
│   ├── 2M16.wav
│   ├── 2M17.wav
│   ├── 2M18.wav
│   ├── 2M19.wav
│   ├── 2M2.wav
│   ├── 2M20.wav
│   ├── 2M21.wav
│   ├── 2M22.wav
│   ├── 2M23.wav
│   ├── 2M24.wav
│   ├── 2M25.wav
│   ├── 2M26.wav
│   ├── 2M27.wav
│   ├── 2M28.wav
│   ├── 2M29.wav
│   ├── 2M3.wav
│   ├── 2M30.wav
│   ├── 2M31.wav
│   ├── 2M32.wav
│   ├── 2M33.wav
│   ├── 2M38.wav
│   ├── 2M39.wav
│   ├── 2M4.wav
│   ├── 2M40.wav
│   ├── 2M5.wav
│   ├── 2M6.wav
│   ├── 2M7.wav
│   ├── 2M8.wav
│   ├── 2M9.wav
│   ├── 3M1.wav
│   ├── 3M10.wav
│   ├── 3M12.wav
│   ├── 3M13.wav
│   ├── 3M14.wav
│   ├── 3M17.wav
│   ├── 3M18.wav
│   ├── 3M19.wav
│   ├── 3M2.wav
│   ├── 3M20.wav
│   ├── 3M21.wav
│   ├── 3M22.wav
│   ├── 3M24.wav
│   ├── 3M26.wav
│   ├── 3M27.wav
│   ├── 3M29.wav
│   ├── 3M3.wav
│   ├── 3M30.wav
│   ├── 3M32.wav
│   ├── 3M33.wav
│   ├── 3M4.wav
│   ├── 3M7.wav
│   ├── 3M8.wav
│   ├── 4M10.wav
│   ├── 4M11.wav
│   ├── 4M12.wav
│   ├── 4M13.wav
│   ├── 4M14.wav
│   ├── 4M15.wav
│   ├── 4M17.wav
│   ├── 4M18.wav
│   ├── 4M19.wav
│   ├── 4M20.wav
│   ├── 4M21.wav
│   ├── 4M22.wav
│   ├── 4M24.wav
│   ├── 4M25.wav
│   ├── 4M28.wav
│   ├── 4M29.wav
│   ├── 4M3.wav
│   ├── 4M31.wav
│   ├── 4M32.wav
│   ├── 4M4.wav
│   ├── 4M5.wav
│   ├── 4M6.wav
│   ├── 4M8.wav
│   ├── 4M9.wav
│   ├── 5M1.wav
│   ├── 5M10.wav
│   ├── 5M11.wav
│   ├── 5M12.wav
│   ├── 5M13.wav
│   ├── 5M14.wav
│   ├── 5M17.wav
│   ├── 5M18.wav
│   ├── 5M23.wav
│   ├── 5M24.wav
│   ├── 5M3.wav
│   ├── 5M4.wav
│   ├── 5M5.wav
│   ├── 5M6.wav
│   ├── 5M8.wav
│   ├── 5M9.wav
│   ├── 6M1.wav
│   ├── 6M11.wav
│   ├── 6M13.wav
│   ├── 6M15.wav
│   ├── 6M2.wav
│   ├── 6M3.wav
│   ├── 6M4.wav
│   ├── 6M5.wav
│   ├── 6M6.wav
│   ├── 6M7.wav
│   ├── 6M8.wav
│   ├── 7M1.wav
│   ├── 7M11.wav
│   ├── 7M14.wav
│   ├── 7M15.wav
│   ├── 7M17.wav
│   ├── 7M18.wav
│   ├── 7M19.wav
│   ├── 7M2.wav
│   ├── 7M20.wav
│   ├── 7M21.wav
│   ├── 7M22.wav
│   ├── 7M23.wav
│   ├── 7M28.wav
│   ├── 7M30.wav
│   ├── 7M31.wav
│   ├── 7M32.wav
│   ├── 7M33.wav
│   ├── 7M34.wav
│   ├── 7M4.wav
│   ├── 7M5.wav
│   ├── 7M6.wav
│   ├── 7M7.wav
│   ├── 7M8.wav
│   ├── 8M1.wav
│   ├── 8M10.wav
│   ├── 8M11.wav
│   ├── 8M12.wav
│   ├── 8M13.wav
│   ├── 8M14.wav
│   ├── 8M15.wav
│   ├── 8M16.wav
│   ├── 8M18.wav
│   ├── 8M19.wav
│   ├── 8M2.wav
│   ├── 8M20.wav
│   ├── 8M21.wav
│   ├── 8M22.wav
│   ├── 8M24.wav
│   ├── 8M26.wav
│   ├── 8M28.wav
│   ├── 8M29.wav
│   ├── 8M30.wav
│   ├── 8M31.wav
│   ├── 8M38.wav
│   ├── 8M39.wav
│   ├── 8M4.wav
│   ├── 8M40.wav
│   ├── 8M41.wav
│   ├── 8M7.wav
│   ├── 9M1.wav
│   ├── 9M10.wav
│   ├── 9M11.wav
│   ├── 9M13.wav
│   ├── 9M14.wav
│   ├── 9M15.wav
│   ├── 9M17.wav
│   ├── 9M18.wav
│   ├── 9M19.wav
│   ├── 9M2.wav
│   ├── 9M20.wav
│   ├── 9M21.wav
│   ├── 9M22.wav
│   ├── 9M23.wav
│   ├── 9M24.wav
│   ├── 9M25.wav
│   ├── 9M3.wav
│   ├── 9M5.wav
│   ├── 9M6.wav
│   ├── 9M7.wav
│   ├── 9M8.wav
│   └── 9M9.wav
├── Saves
│   ├── empty
│   ├── sword1.000
│   ├── sword1.001
│   ├── sword1.002
│   ├── sword1.003
│   ├── sword1.004
│   ├── sword1.005
│   ├── sword1.006
│   └── sword1.007
├── Smackshi
│   ├── CANDLE.dxa
│   ├── CANDLE.fla
│   ├── CREDITS.dxa
│   ├── CREDITS.fla
│   ├── FERRARI.dxa
│   ├── FERRARI.fla
│   ├── FINALE.dxa
│   ├── FINALE.fla
│   ├── GEODROP.dxa
│   ├── GEODROP.fla
│   ├── GRAVE.dxa
│   ├── GRAVE.fla
│   ├── HISTORY.dxa
│   ├── HISTORY.fla
│   ├── INTRO.dxa
│   ├── INTRO.fla
│   ├── IRELAND.dxa
│   ├── IRELAND.fla
│   ├── LADDER.dxa
│   ├── LADDER.fla
│   ├── MONTFCON.dxa
│   ├── MONTFCON.fla
│   ├── RIVER.dxa
│   ├── RIVER.fla
│   ├── SEWER.dxa
│   ├── SEWER.fla
│   ├── SPANISH.dxa
│   ├── SPANISH.fla
│   ├── STEPS.dxa
│   ├── STEPS.fla
│   ├── TAPESTRY.dxa
│   ├── TAPESTRY.fla
│   ├── TRUCK.dxa
│   ├── TRUCK.fla
│   ├── VULTURE.dxa
│   ├── VULTURE.fla
│   ├── WELL.dxa
│   └── WELL.fla
└── Speech
    └── speech.clu

6 directories, 315 files
```

(Ignore the Saves directory, these are my old game saves.)

**Pull request merged**